.github/workflows/e2e_test.yaml

```diff
@@ -270,7 +270,9 @@ jobs:
           [[ -z "${http_proxy}" && -z "${HTTP_PROXY}" ]] \
             || cat /home/ubuntu/.docker/config.json | grep httpProxy
       - name: Install microk8s
-        run: sudo snap install microk8s --classic
+        # TMP: Pin version of microk8s to older version due to issues with running microk8s in LXD
```

Comment:
Is this about running unconfined lxd containers or something else? I suggest adding a link to an issue or a clarifying comment what the problem is (although I think there should be an open issue we can link to).